pkg/client/request.go

```diff
@@ -236,7 +236,7 @@ func (r *Request) ParseSelectorParam(paramName, item string) *Request {
 	if r.err != nil {
 		return r
 	}
-	sel, err := labels.ParseSelector(item)
+	sel, err := labels.Parse(item)
```

Comment:
We need to use ParseSelector when paramName = "fields" and Parse when paramName = "labels".